apps/site/lib/site_web/templates/layout/app.html.eex

```diff
@@ -18,7 +18,14 @@
     if @conn.request_path == "/org" || String.slice(@conn.request_path, 0..4) == "/org/" do %>
       <meta name="robots" content="noindex, nofollow">
     <% end %>
-
+    <%= # hide any page in /charlie directory from search engines
+    if @conn.request_path == "/charlie" || String.slice(@conn.request_path, 0..4) == "/charlie/" do %>
+      <meta name="robots" content="noindex, nofollow">
+    <% end %>
+    <%= # hide any page in /policies/terms-use-charlie directory from search engines
+    if @conn.request_path == "/policies/terms-use-charlie" || String.slice(@conn.request_path, 0..4) == "/policies/terms-use-charlie/" do %>
```

Comment:
Both of these `slice` calls need to be changed.  The function as written will return `String.slice("/charlie/", 0..4)` -> `/char` and `String.slice("/policies/terms-use-charlie/", 0..4)` -> `/poli`.  This will fail to match on the desired string